README.md

```diff
@@ -61,7 +61,7 @@ Generated content in a rule doc (everything above the marker comment):
 
 💼 This rule is enabled in the following configs: `all`, ✅ `recommended`.
 
-🔧 This rule is automatically fixable by the `--fix` [CLI option](https://eslint.org/docs/latest/user-guide/command-line-interface#--fix).
+🔧 This rule is automatically fixable by [the `--fix` CLI option](https://eslint.org/docs/user-guide/command-line-interface#--fix).
```

Comment:
Can we exclude `the` from the link? I don't think that needs to be part of it.

I actually got some similar feedback in this comment: https://github.com/sindresorhus/eslint-plugin-unicorn/pull/1930#discussion_r990917280